src/SfxWeb/src/app/modules/backup-restore/storage-form/storage-form.component.html

```diff
@@ -38,24 +38,32 @@
             <dd>
                 <input type="text" class="input-flat" formControlName="Path" name="Path">
             </dd>
+            <dt>IsPrimaryCredsEmpty</dt>
```

Comment:
is there a better wording here, to make it more obvious

Reply:
Thanks. I have made the necessary changes